lib/loaders/followed_artist.js

```diff
@@ -0,0 +1,26 @@
+import DataLoader from "dataloader"
+import gravity from "../apis/gravity"
+import { toKey } from "../helpers"
+import { map, find, extend, first } from "lodash"
+
+const followedArtistLoader = new DataLoader(
+  keys => {
+    const keyObjs = map(keys, JSON.parse)
+    const { accessToken } = first(keyObjs)
+    const ids = map(keyObjs, "id")
+    const path = toKey("me/follow/artists", {
+      artists: ids,
+    })
+    return gravity(path, accessToken).then(({ body }) => {
+      const parsedResults = map(ids, id => {
+        const match = find(body, follow => follow.artist.id === id)
+        if (match) return extend(match.artist, { is_followed: true })
```

Comment:
Cool, Any reason to return the full artist/id, since we just pluck out the `is_followed` part anyway?